layers/+lang/julia/README.org

```diff
@@ -31,25 +31,25 @@ Warning: enabling the LSP functionality within this package can cause emacs to
 hang for significant lengths of time when opening a Julia file. See tip below on
 using PackageCompiler.jl to mitigate this issue.
 
-This layer can be used with [[https://github.com/JuliaEditorSupport/LanguageServer.jl][ =LanguageServer.jl= ]] and emacs [[https://github.com/emacs-lsp/lsp-mode][ =lsp-mode= ]] to provide
+This layer can be used with [[https://github.com/JuliaEditorSupport/LanguageServer.jl][ =LanguageServer.jl=]] and emacs [[https://github.com/emacs-lsp/lsp-mode][ =lsp-mode=]] to provide
```

Comment:
Not to be a drag, but shouldn't it also remove leading spaces? e.g. `[=LanguagerServer.jl=]` and `[=lsp-mode=]` instead of `[ =LanguagerServer.jl=]` and `[ =lsp-mode=]`? (note the space between `[` and `=`]